### ISSUE TITLE
fatal error container start

### ISSUE THREAD

**Issue by mfumagalli68** [ffeldhaus/docker-wireshark#4]
Hi thanks for sharing this.

I got an error when running 

`docker run --restart unless-stopped --name wireshark  --cap-add NET_ADMIN ffeldhaus/wireshark`

```
2020-12-15 13:28:14,884 Error: cannot enable SSH socket upgrades:
2020-12-15 13:28:14,884  No module named 'paramiko'
2020-12-15 13:28:14,884 created tcp socket '0.0.0.0:14500'
2020-12-15 13:28:14,886 cannot access python uinput module:
2020-12-15 13:28:14,886  No module named 'uinput'
_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.
2020-12-15 13:28:16,107 created unix domain socket '/run/user/1000/xpra/****myhost*****-0'
2020-12-15 13:28:16,184 pointer device emulation using XTest
2020-12-15 13:28:16,320 serving html content from '/usr/share/xpra/www'
2020-12-15 13:28:16,364 pulseaudio not started: 'pulseaudio' command not found
Warning: failed to import GStreamer 1.x:
 Namespace Gst not available
2020-12-15 13:28:16,549 Error: failed to query sound subsystem:
2020-12-15 13:28:16,549  query did not return any data

(Xpra:7): Gtk-CRITICAL **: 13:28:16.605: gtk_widget_realize: assertion 'widget->priv->anchored || GTK_IS_INVISIBLE (widget)' failed
2020-12-15 13:28:16,611 Error: failed to import the virtual video module:
2020-12-15 13:28:16,612  No module named '_dbus_bindings'
2020-12-15 13:28:16,616 started command 'wireshark --fullscreen' with pid 74
2020-12-15 13:28:16,617 Warning: cannot watch for application menu changes without pyinotify:
2020-12-15 13:28:16,617  No module named 'pyinotify'
wireshark: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
2020-12-15 13:28:16,621 Warning: cannot use application menu data:
2020-12-15 13:28:16,622  no python-xdg module
2020-12-15 13:28:16,892 child 'wireshark --fullscreen' with pid 74 has terminated
2020-12-15 13:28:16,892 xpra is ready.
2020-12-15 13:28:16,893 all children have exited and --exit-with-children was specified, exiting
2020-12-15 13:28:16,896 xpra GTK3 X11 server is terminating
2020-12-15 13:28:16,896 xpra GTK3 X11 version 4.0.4-r27557 64-bit
2020-12-15 13:28:16,930 waiting for initialization thread to complete
2020-12-15 13:28:16,996  uid=1000 (xpra), gid=106 (xpra)
2020-12-15 13:28:16,996  running with pid 7 on Linux unknown unknown unknown
2020-12-15 13:28:17,070 31.4GB of system memory
2020-12-15 13:28:17,990 OpenGL is supported on display ':0'
2020-12-15 13:28:17,991  using 'llvmpipe (LLVM 10.0.0, 256 bits)' renderer
2020-12-15 13:28:17,993 killing xvfb with pid 14
2020-12-15 13:28:17,993 closing tcp socket '0.0.0.0:14500'
2020-12-15 13:28:17,993 removing socket '/run/user/1000/xpra/*******myhost****-0'
Gdk-Message: 13:28:17.998: Xpra: Fatal IO error 0 (Success) on X server :0.

```
I simpy pull the image from Docker Hub (https://hub.docker.com/r/ffeldhaus/wireshark/) and run the above command.

Docker version 19.03.5, build 633a0ea
Host OS: Centos 7.

**Comment by ffeldhaus** [comment]
The issue is, that wireshark fails to start and the error is in the logs above, but pretty well hidden:
    
    wireshark: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory

I will have a look at the issue in the next days and check if I can recreate it. I usually test and run this on a Ubuntu or Debian host OS, where I just verified that there are no issues with Docker 19.03.14.

**Comment by ffeldhaus** [comment]
The issue is fixed in the latest version (1.1 or latest). It is releated to the libQt5Core.so.5 having the wrong ELF section for Centos. The workaround provided here solves the issue as it is not WSL specific: https://github.com/microsoft/WSL/issues/3023
